Server/BackEndMonitoringServer/src/CProcessesInfoMonitoring.cpp

```diff
@@ -0,0 +1,73 @@
+#include "stdafx.h"
+
+#include "CContainerOfProcesses.h"
+#include "CJSONFormatterProcess.h"
+#include "CThreadSafeVariable.h"
+#include "CJSONFormatSaver.h"
+#include "CProcessInfo.h"
+#include "CProcessesInfoMonitoring.h"
+#include "CEvent.h"
+
+CProcessesInfoMonitoring::CProcessesInfoMonitoring(
+	std::chrono::duration<int> pause_duration,
+	std::string path_to_file,
+	EMemoryConvertType count_type,
+	CEvent& stop_event,
+	CThreadSafeVariable<CJSONFormatterProcess>& json_formatter) :
+		m_container(pause_duration, path_to_file, count_type),
+		m_json_formatter(json_formatter),
+		IHardwareInfoMonitoring(stop_event),
+		m_is_initialized(false)
+{ }
+
+bool CProcessesInfoMonitoring::Initialize()
+{
+	bool success;
```

Comment:
A very bad piece of code maybe there should be:
const bool cb_success = m_container.Initialize() && m_container.TryToUpdateCurrentStatus();

Reply:
Fixed